.env.regtest

```diff
@@ -0,0 +1,23 @@
+NEXT_PUBLIC_ENV=regtest
+NEXT_PUBLIC_RIF_WALLET_SERVICES=http://localhost:3000/mock
+NEXT_PUBLIC_EXPLORER=/mock
+NEXT_PUBLIC_BUILD_ID=
+
+NEXT_PUBLIC_RIF_ADDRESS=0x19f64674d8a5b4e652319f5e239efd3bc969a1fe
+NEXT_PUBLIC_STRIF_ADDRESS=0xAF17f7A0124E9F360ffA484b13566b041C0f5023
+NEXT_PUBLIC_GOVERNOR_ADDRESS=0xEc6bd0C8117b74904849af2CED73f30090DB6cd1
+NEXT_PUBLIC_EA_NFT_ADDRESS=0x687E04Bb759B3A010eb797301E5D1D05e135E90f
+NEXT_PUBLIC_MULTICALL_ADDRESS=0xcA11bde05977b3631167028862bE2a173976CA11
+NEXT_PUBLIC_GRANTS_BUCKET_ADDRESS=0xfaca664c661af7e0e630c8f92b401012cd2a30ef
+NEXT_PUBLIC_GRANTS_ACTIVE_BUCKET_ADDRESS=0x2217E4d3Ae0A6E30075D1B5a7b8C1520E8009f49
+NEXT_PUBLIC_GROWTH_BUCKET_ADDRESS=0x420f39968009a0e8693a0e1bba633cd99eaffb9b
+NEXT_PUBLIC_GENERAL_BUCKET_ADDRESS=0x72Ed7d7b7835Ad62B1f9b6280bAd62618aA71461
+NEXT_PUBLIC_CHAIN_ID=33
+
+REGTEST_URL=http://localhost:4444
+# CR-related env variables
+NEXT_PUBLIC_SIMPLIFIED_REWARD_DISTRIBUTOR_ADDRESS=0x2E8924AAa43A9544A058E6f97D2d7a80a5266c28
+NEXT_PUBLIC_CYCLE_DURATION_IN_DAYS=7
+NEXT_PUBLIC_FIRST_CYCLE_START_DATE_ISO="1970-01-01T00:00:00Z"
+
+NEXT_PUBLIC_ENV_DATA_URL="https://raw.githubusercontent.com/antomor/public-access-json/main/data.regtest.json"
```

Comment:
This needs to be changed

Reply:
is this a public repo now?

Reply:
Yes, I'd just create a separate PR for the data, so we can update the value here